platform/broadcom/sonic-platform-modules-mitac/debian/sonic-platform-mitac-ly1200-32x.init

```diff
@@ -13,6 +13,11 @@
 
 case "$1" in
 start)
+    eval sonic_version=$(cat /etc/sonic/sonic_version.yml | grep build_version | cut -f2 -d" ")
+    if [ -f /host/image-$sonic_version/platform/firsttime ]; then
+        update-rc.d xcvr_servd defaults >/dev/null
+    fi
```

Comment:
this needs to be moved into postinst script, it should not be in the init script.

Reply:
postinst will automatically add `depmod -a`?

Reply:
I will move this to postinst.